ip-interfaces.lisp

```diff
@@ -30,6 +30,7 @@
   address-family)
 
 (defun bytes (pointer count)
+  (declare (optimize (debug 3)))
```

Comment:
I assume we don't want this to be a permanent change? (Similarly with other occurrences in the file.)

Reply:
Yes. I was in such a hurry to fix the broken `-value` calls that I didn't notice these still in there.
Thanks.